tests/test_cloud_function.py

```diff
@@ -20,3 +20,70 @@ def test_last_date_db(mocker):
                 return_value = df_test_last_date
                 )
     assert last_date_db() == '2022-06-10T17:57:21Z'
+
+
+json_test_next = {'data': [{
+                'lang': 'en',
+                'created_at': '2022-07-06T13:53:07.000Z',
+                'id': '1544680846883528706',
+                'text': '“This weekend won’t be smooth sailing” says Championship Leader Max Verstappen #Formula1 #F1 #F1News #Motorsport https://t.co/ncnzaoqvcd'
+                    }],
+            'meta': {'newest_id': '1544680846883528706',
+                'oldest_id': '1544679599849623552',
+                'result_count': 100,
+                'next_token': 'b26v89c19zqg8o3fpz2m15nlsnmmd4ev3codr28uvwzgd'
+                    }
+            }
+
+json_test_no_next = {'data': [{
```

Comment:
FAKE_TWITTER_API_RESPONSE_WITH_NO_NEXT

Reply:
Changed name of variable to FAKE_TWITTER_API_RESPONSE_WITH_NO_NEXT